.gitignore

```diff
@@ -39,3 +39,7 @@ yarn-error.log*
 
 # SQLite
 prisma/dev.*
+
+# NPM
```

Comment:
Why do we want to ignore the package-lock files? Looks like having it would be helpful to keep track of the current working directory tree:https://stackoverflow.com/questions/48524417/should-the-package-lock-json-file-be-added-to-gitignore

Reply:
+1, never ignore package-lock, and certainly not package.json